iamctl/pkg/utils/setup.go

```diff
@@ -47,8 +48,18 @@ type ServerConfigs struct {
 }
 
 type ToolConfigs struct {
-	KeywordMappings     map[string]interface{} `json:"KEYWORD_MAPPINGS"`
 	AllowDelete         bool                   `json:"ALLOW_DELETE"`
+	Exclude             []string               `json:"EXCLUDE"`
+	IncludeOnly         []string               `json:"INCLUDE_ONLY"`
+	ExcludeSecrets      bool                   `json:"EXCLUDE_SECRETS"`
+	ApplicationConfigs  map[string]interface{} `json:"APPLICATIONS"`
+	IdpConfigs          map[string]interface{} `json:"IDENTITY_PROVIDERS"`
+	ClaimDialectConfigs map[string]interface{} `json:"CLAIM_DIALECTS"`
```

Comment:
```suggestion
	ClaimConfigs map[string]interface{} `json:"CLAIMS"`
```
It will be confusing for the user to use the term ` claim dialects`. Let's change line 65 as well. Check for other places.